### ISSUE TITLE
Hyper should fallback to default config when not specified in `~/.hyper.js`

### ISSUE THREAD

**Issue by sindresorhus** [zeit/hyper#1588]
My `~./hyper.js` is:

```js
'use strict';
module.exports = {
	config: {
		fontSize: 16,
		fontFamily: 'Menlo'
	},
	plugins: [
		'hyper-snazzy'
	]
};
```

Since I deleted default config like the `css` option, that property is now `undefined` in `decorateConfig` for plugins. Hyper should provide default values in the core for all config properties.

For now, I'll have to specify an empty `css: ''` option so plugins/themes using `config.css` doesn't break.

**Comment by JessDF** [comment]
Hello, I was wondering - Is anyone is working on this bug at the moment?

**Comment by ppot** [comment]
Could be integrated into #1509

**Comment by okonek** [comment]
I created PR

**Comment by chabou** [comment]
Fixed by #2126 
Thank you so much @okonek ❤️
